rfcs/components/moving-from-focus-zone-to-tabster.md

```diff
@@ -0,0 +1,155 @@
+# Moving from `FocusZone` to `Tabster`
+
+---
+
+Contributors: @khmakoto
+
+## Problem statement
+
+We currently use `FocusZone` for focus management in both v0 and v8. For our converged components we are trying out a different approach by using `Tabster`.
+
+The reason for doing this is use a framework-agnostic solution (as opposed to the React-centric `FocusZone`) that operates at the DOM level and has no external runtime dependencies.
```

Comment:
Tabster seems to be, and I quote, "_in a work-in-progress proof-of-concept state_".
Are the benefits exclusively having a framework-agnostic solution and no dependencies or are there features `FocusZone` does not have that we need?
Personally, I would rather bet in a new less mature solution if we explicitly need it.

Reply:
Content added from #17129 , that gives a more detailed look on the functionalities that tabster provides.

Teams currently uses Tabster

Reply:
> Tabster seems to be, and I quote, "_in a work-in-progress proof-of-concept state_".
That is a note for third parties. It is work-in-progress by us and for us with our specific requirements and accessibility experience in mind.

Reply:
Browsers handle tabbing on the DOM level, assistive technologies (such as screen readers) operate on the DOM level. The focus management is the basic thing which should be solved on the DOM level, and without messing with something like breaking React components encapsulation. Which you would have to do to, for example, get the list of focusable elements in a particular part of the application or (which is even more fun) outside of the particular part of the application. And integrate that all with the React components life cycle.